spec/features/user_sees_dashboard_functionality_when_signed_in_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'As a logged in user' do
+  before :each do
+    visit root_path
+    stub_omniauth
+    click_link 'Sign in with Google'
+
+    expect(current_path).to eq(dashboard_path)
+    expect(page).to have_content('Happy')
+    expect(page).to have_content('Gilmore')
+  end
+
+  it "shows options for users via dropdown menus before they shake it up" do
```

Comment:
This is testing to make sure the information we want displayed on the user dashboard is displayed when the user has logged in.